appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockCircularProgress.java

```diff
@@ -0,0 +1,46 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2009-2011 Google, All Rights reserved
+// Copyright 2011-2012 MIT, All rights reserved
```

Comment:
@barreeeiroo These copyright years do not seem correct. Since this was a component added by Kodular I would expect there would Kodular and MIT noted here.